recipes/libmagic/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "5.45":
+    url: "https://github.com/file/file/archive/FILE5_45.tar.gz"
```

Comment:
Please, use http://ftp.astron.com/pub/file/file-5.45.tar.gz as main source of download and GH as mirror. It's described both in GH repo and https://www.darwinsys.com/file/ that GH is a mirror.